api/v1beta2/kyma_types.go

```diff
@@ -387,24 +387,42 @@ func (kyma *Kyma) IsBeta() bool {
 
 type AvailableModule struct {
 	Module
-	Enabled bool
-	Valid   bool
+	Enabled         bool
+	ValidationError string
+}
+
+func (a AvailableModule) IsValid() bool {
```

Comment:
I don't recommend this way to indicate `IsValid`,  if you declare `ValidationError` as `Error`, then you can return `a.ValidationError == nil`

Reply:
I think that external API is more important than internal implementation. Because we have this type in the `api/v1beta2` package, controller-gen is trying to process it and fails. I don't know to disable this other than moving this type to some other package.

Reply:
`IsValid()` is removed.